var/docs/docker.md

```diff
@@ -0,0 +1,52 @@
+# Docker
+
+We bundle a simple docker that can be used for test purposes.  Either pull it from docker hub or build it.
+
+## Build the docker:
+
+    docker build -t $USER/kimai2 .
+
+## Run the docker
+
+    docker run -ti -p 8001:8001 --name kimai2 --rm $USER/kimai2
+
+There is an admin user with the password admin but no fixtures installed. To install the fixtures:
+
+    docker exec kimai2 bin/console kimai:reset-dev
```

Comment:
As these commands only work after an image was started, shall we explain the demonize `-d` flag?

Reply:
Yes.  But we are straying onto writing a docker howto.  I've added the words "once started" to the Running commands section but I think we need to assume a certain level of ability.

Reply:
True. I just experienced time after time in all the years of managing open source projects, that people will execute stuff from the docu step-by-step, then something doesn't work and they come to ask here with the question instead of reading the docker docs. But you are right, lets wait and see what happens